frontend/src/components/MyDatesPicker.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import ReactDOM from "react-dom";
+import DatePicker from "react-date-picker";
+import DatePicker from "react-date-picker";
+import Col from "react-bootstrap/Col";
+import Row from "react-bootstrap/Row";
+import { ResponsiveContainer } from "recharts";
+
+export default class MyDatesPicker extends React.Component {
+  state = {
+    date: [new Date(), new Date()],
+    dateTwo: [new Date(), new Date()],
```

Comment:
Can we change these variable names to be more descriptive (eg. lockdownDate and firstRestrictionsDate)?

Reply:
yes, this is being refactored in PR #77, they might become instances of the InfoPanel class, given that this information inherits to several of the sub components.